### PR TITLE
fix: full page story with global header

### PR DESCRIPTION
**Pull request by lee-chase**
A global header does not work with full page create 

#### What did you change?

- Full page edit to remove header height calculations and use display grid.
- Added story to demonstrate

E.g.
image

#### How did you test and verify your work?

Storybook

#### Next
May need to raise issue for other full page e.g. edit?

### CODE REVIEW

File: packages/ibm-products-styles/src/components/CreateFullPage/_create-full-page.scss [hunk 1]
```diff
@@ -21,8 +21,15 @@
 $block-class: #{c4p-settings.$pkg-prefix}--create-full-page;
 $step-block-class: #{c4p-settings.$pkg-prefix}--create-full-page__step;
 
-.#{$block-class} .#{$block-class} {
+.#{$block-class}.#{$block-class} {
```
Comment (paul-balchin-ibm) [inline packages/ibm-products-styles/src/components/CreateFullPage/_create-full-page.scss:24]: Duplicate class names?

Comment (lee-chase) [reply packages/ibm-products-styles/src/components/CreateFullPage/_create-full-page.scss:24]: @paul-balchin-ibm I didn't add the duplicate names, just made them functional assuming they were supposed to.

Removed the duplicate.

**Review by davidmenendez — APPROVED** [review]

File: packages/ibm-products-styles/src/__tests__/__snapshots__/styles.test.js.snap [hunk 2]
```diff
@@ -1908,8 +1908,14 @@ p.c4p--about-modal__copyright-text:first-child {
   letter-spacing: var(--cds-label-01-letter-spacing, 0.32px);
 }
 
-.c4p--create-full-page .c4p--create-full-page {
+.c4p--create-full-page {
+  display: grid;
+  height: 100%;
   color: var(--cds-text-primary, #161616);
+  grid-template-rows: minmax(auto, 100%);
```
Comment (baraaabdelsalam) [inline packages/ibm-products-styles/src/__tests__/__snapshots__/styles.test.js.snap:1915]: Hello @lee-chase 

This doesn't work on some browsers, causing some height issues as you can see from the screenshot.

Screenshot 2024-09-02 at 15 40 42

Comment (baraaabdelsalam) [reply packages/ibm-products-styles/src/__tests__/__snapshots__/styles.test.js.snap:1915]: I have a fix available by using `auto minmax(min-content, 100%)` instead... if there are no objections, I can contribute that.

**Pull request merged**